modules/ROOT/partials/module-loading/webpack-dev-dependencies.adoc

```diff
@@ -1,15 +1,15 @@
-{% if is_zip_install == true %}
-1. Add the following development dependencies to the project.
-
-- [`webpack`](https://www.npmjs.com/package/webpack)
-- [`webpack-cli`](https://www.npmjs.com/package/webpack-cli)
-- [`mini-css-extract-plugin`](https://www.npmjs.com/package/mini-css-extract-plugin)
-- [`html-webpack-plugin`](https://www.npmjs.com/package/html-webpack-plugin)
-- [`css-loader`](https://www.npmjs.com/package/css-loader)
+ifeval::[{is_zip_install} == true]
+. Add the following development dependencies to the project.
 
+* [`webpack`](https://www.npmjs.com/package/webpack)
+* [`webpack-cli`](https://www.npmjs.com/package/webpack-cli)
+* [`mini-css-extract-plugin`](https://www.npmjs.com/package/mini-css-extract-plugin)
+* [`html-webpack-plugin`](https://www.npmjs.com/package/html-webpack-plugin)
+* [`css-loader`](https://www.npmjs.com/package/css-loader)
++
 For example:
-
-[source,sh]
++
+[source,sh,indent=0]
```

Comment:
Why was the `indent=0` needed here?

Reply:
I tried to remove it and it work in the same way, I probably had added it for a try and forgot to remove it, sorry, I'll remove

Reply:
done